scripts/zones/The_Garden_of_RuHmet/mobs/Jailer_of_Faith.lua

```diff
@@ -18,10 +18,10 @@ local openFlower = function(mob)
     mob:setMod(xi.mod.SLASH_SDT, 1250)
     mob:setMod(xi.mod.PIERCE_SDT, 1250)
     mob:setMod(xi.mod.IMPACT_SDT, 1250)
-    mob:setMod(xi.mod.DMGPHYS, 112)
-    mob:setMod(xi.mod.DMGMAGIC, 112)
-    mob:setMod(xi.mod.DMGRANGE, 112)
-    mob:setMod(xi.mod.DMGBREATH, 112)
+    mob:setMod(xi.mod.DMGPHYS, 11200)
+    mob:setMod(xi.mod.DMGMAGIC, 11200)
+    mob:setMod(xi.mod.DMGRANGE, 11200)
+    mob:setMod(xi.mod.DMGBREATH, 11200)
```

Comment:
This could be bundled up into a single mod named `DMG` instead of all 4 damage types.

Reply:
Changed after reworking damage taken applications.